governance/SDK.md

```diff
@@ -0,0 +1,172 @@
+
+The new maintainer must show the good will to keep the project healthy and
+community could ask for an eventual plan to improve the quality of the project.
+
+As soon as the new maintainer is identified, the community can proceed to vote
+to handover the project using the
+[Asynchronous voting process](#asynchronous-voting-process). If more than one
+new maintainer wants to handover the project, separate voting process must be
+done for all of them, and both of them must follow the same voting criteria (and
+not the ones from [New maintainers section](#new-maintainers)).
+
+The voting criteria are:
+
+- 1 week to vote
+- 2/3 of the agreement to accept the vote
+- All _sdk maintainers_ are entitled to vote
+
+### Archive a project
+
+It may happen that, if a project is not following the criteria 1, 2, 4, 5, 6, 7,
+8, the community might decide to archive the project. Prior to archiving the
+project, the community should consider first to search a new maintainer to
+[handover the project](#handover-to-a-new-maintainergroup-of-maintainers).
+
+If no new maintainer is found, the community can proceed to archive the project
+using the [Asynchronous voting process](#asynchronous-voting-process).
+
+The voting criteria are:
+
+- 2 weeks to vote
+- 2/3 of the agreement to accept the vote
+- All _sdk maintainers_ are entitled to vote
+
+## Asynchronous voting process
+
+In order to ensure the democratic management of debates in the community, we
+define an asynchronous voting process using GitHub issues.
+
+The voting is defined by:
+
+- The proposed community change to vote
+- The set of people/groups entitled to vote
+- A voting period
+- The percentage of agreements required to accept the change
+- A member of the community entitled to manage the vote, called voting manager.
+  The voting manager MUST also be part of people entitled to vote.
+
+To start a vote, the voting manager opens an issue in the
+[CloudEvents Spec](https://github.com/cloudevents/spec/issues) with:
+
+- The binary question which explains the change
+- The tags to the entitled people/groups to vote (depending on the question)
+- The expiration date of the vote
+- The percentage of agreements to accept the change
+
+For example:
+
+> Question: Do you want to add "Paolo Rossi" @paolorossi as _sdk-go-maintainer_?
+>
+> Entitled to vote:
+>
+> - @sdk-go-maintainers
+>
+> Voting ends: 24th June 2020 12:00 CEST
+>
+> Required agreement: 50% + 1
+
+Then every person entitled can vote, up to the expiration date, commenting on
+the issue with a `+1` to express agreement or `-1` to express disagreement.
+Every comment except `+1` and `-1` won't be taken in account, because we expect
+that the voting process comes only after a debate on the change, developed on
+the community channels and Serverless WG/CloudEvents SDK meetings. In order to
+ensure visibility to the voting, people should be remembered to vote during
+Serverless WG meetings and community channels (Slack, ...).
```

Comment:
What about using something like [gh-polls](https://gh-polls.com/)? I find going through and counting up +1 comments to be tedious and error prone - especially when you start asking yourself, does :+1: on the issue count as a yes vote.

Reply:
> I find going through and counting up +1 comments to be tedious and error prone - especially when you start asking yourself, does +1 on the issue count as a yes vote.

+1 is +1, i don't think it leaves too much room for interpretations :smile: I would prefer to avoid including another tool, because if we want to adopt this solution in other repos then we need to configure the tool in other repos and so on. It doesn't seem to me it adds enough value and +1 seems pretty straightforward to me. For the beginning IMO sticking with +1/-1 it's ok